app/assets/javascripts/admin/calendar/dependencies.js

```diff
@@ -0,0 +1,2 @@
+//= require moment
+//= require fullcalendar
```

Comment:
Why have these dependencies in a separate file? You can put them at the top of `calendar.js.coffee`—just change the syntax to `#=` for coffeescript.